### ISSUE TITLE
Support for Objective-C?

### ISSUE THREAD

**Issue by AlexChesters** [github-linker/chrome-extension#55]
Currently Objective-C imports like [this](https://github.com/AlexChesters/ukpd-ios/blob/master/ukpd/ukpd/view-controllers/MainTableViewController.m#L9) try to find an NPM module with name of the class.

**Comment by stefanbuck** [comment]
No, Objective-C is not supported yet. The extension use a fuzzy method to replace the links. I'm working to improve it with the next major version.

**Comment by shakeelmohamed** [comment]
👍  for C++ as well
